src/encrypter.ts

```diff
@@ -128,7 +128,11 @@ export class Encrypter {
     if ('kModuleError' in mongodbClientEncryption) {
       throw new MongoMissingDependencyError(
         'Auto-encryption requested, but the module is not installed. ' +
-          'Please add `mongodb-client-encryption` as a dependency of your project'
+          'Please add `mongodb-client-encryption` as a dependency of your project',
+        {
+          cause: mongodbClientEncryption['kModuleError'].cause,
```

Comment:
```suggestion
          cause: mongodbClientEncryption['kModuleError'],
```

Why not include the entire chain of errors? then we don't lose the full stack

Reply:
kModuleError is already a MissingDependencyError, so we'd end up with a missing dependency error wrapping a missing dependency error wrapping an import error.  The logic here is basically equivalent to:

```
mongodbClientEncryption['kModuleError'].message = 'Please add `mongodb-client-encryption` as a dependency of your project'
throw mongodbClientEncryption['kModuleError']
```

But I didn't want to modify the global `kModuleError` property, so I chose to continue instantiating a new error instead.

Reply:
> we'd end up with a missing dependency error wrapping a missing dependency error wrapping an import error

What's wrong with having that chain of errors though? It is the "truth" in the sense of what a "cause" is supposed to mean and improves the traceability. It is one step away today but that might not always be the case

Reply:
> It is the "truth" in the sense of what a "cause" is supposed to mean

Not really.  The first missing dependency error did not "cause" the second, we're just transforming the error message.

Reply:
True, but the full stack trace is what I'm hoping to preserve here